packages/explain-plan-helper/src/index.ts

```diff
@@ -1,8 +1,11 @@
 import convertExplainCompat from 'mongodb-explain-compat';
+import { getPlannerInfo } from './getPlannerInfo';
+import { getExecutionStats } from './getExecutionStats';
+import type { ExecutionStats } from './getExecutionStats';
 
 const kParent = Symbol('ExplainPlan.kParent');
 
-type Stage = Record<string, any> & { stage: string; [kParent]: Stage | null };
+export type Stage = Record<string, any> & { stage: string;[kParent]: Stage | null };
```

Comment:
also something the linter should catch, I think:
```suggestion
export type Stage = Record<string, any> & { stage: string; [kParent]: Stage | null };
```

Reply:
i got to fix my linter!